app/assets/stylesheets/application.css

```diff
@@ -1,12 +1,19 @@
+@import "carstyling.css";
+@import "pagination.css";
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
   list-style: none;
 }
 
-.App {
+/* .App {
   display: grid;
   grid-template-columns: 1fr 5fr;
   height: 100vh;
-}
```

Comment:
- Kindly uncomment this styling as it allows your navigation and car list to be displayed correctly.

Reply:
On the project structure, here are my suggestions.
- Considering the heading, there is a part @Eternalgratis designed which will be intergrated into to work as a  heading. That means the heading i have, has to be removed.
- Shadow design and 3 cars, the requirements do not restrict from making the customized design as long as you don't represent a very different meaning.

Reply:
Hey, @AlphaNtihinduka thank you for highlighting this, lets's meet to discuss these via a zoom call.